### PR TITLE
Forms Automated Test - LPS-131362 | master

### PR DESCRIPTION
**Pull request by PauloVAS**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSelectFromListField.testcase [hunk 1]
```diff
@@ -87,13 +87,62 @@ definition {
...
+			fieldType = "Select from List");
+
+		FormFields.editOptionsRow(
+			optionName = "Option1",
+			optionValue = "Option 1",
+			rowNumber = "1");
+
+		Form.gotoAdvancedTab();
+
+		FormViewBuilder.setFieldRepeatable();
```
Comment (alinedoleron) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSelectFromListField.testcase:114]: Hi, @PauloVAS. I noticed this macro `FormViewBuilder` belongs to AppBuilder, not to Forms. Please, can you replace this line by `FormFields.enableSwitcher(fieldName = "Repeatable")`. Thanks!

**Review by alinedoleron — CHANGES_REQUESTED** [review]
Hi @PauloVAS, there're two points I've commented on this PR. Please, take a look on them. =) Thanks!

Good job!

**Comment by PauloVAS** [comment]
@alinedoleron, fixed!

**Comment by alinedoleron** [comment]
ci:forward

**Pull request closed without merging**